<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="my_label" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mouse_movement">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>    Mouse Movement</string>
    </property>
   </widget>
   <widget class="QLabel" name="mouse_pressed">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse Pressed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(350 X 350)</string>
    </property>
   </widget>
   <widget class="Line" name="x_axis">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>225</y>
      <width>450</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 85, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="y_axis">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>0</y>
      <width>2</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_axes">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Axes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Draw">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="circle_radius">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>260</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="draw_circle">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Circle</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="draw_line">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>77</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_point1">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>290</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set point 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_point2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>290</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set point 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>my_label</class>
   <extends>QLabel</extends>
   <header location="global">my_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
